db/migrate/20170101214034_add_default_value_to_users_refresh_interval.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultValueToUsersRefreshInterval < ActiveRecord::Migration[5.0]
+  def change
+    change_column_null :users, :refresh_interval, false, 0
```

Comment:
Looks like I had the syntax right, but it didn't do what I thought.  That just changes existing rows to 0.  [The documentation](http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_column_null) even says "Please note the fourth argument does not set a column's default."

I just needed to RTFM